hadoop-tools/hadoop-azure/dev-support/testrun-scripts/testsupport.sh

```diff
@@ -63,6 +64,37 @@ fnsBlobConfigFileCheck() {
   fi
 }
 
+if ! command -v az &> /dev/null
```

Comment:
There is a similar check already present in script. Check `checkdependencies()` method in testsupport.sh file.
But there they do not install, they rely on user to manually install.

Reply:
This way also looks good. Just see the if statement format used there, that looks better.